actors/abi/sector.go

```diff
@@ -209,7 +235,7 @@ type InteractiveSealRandomness Randomness
 
 // Information needed to verify a seal proof.
 type SealVerifyInfo struct {
-	RegisteredProof
+	RegisteredSealProof
```

Comment:
For brevity, I'd mildly prefer `SealProof RegisteredSealProof`, and similarly for the others below.

Reply:
Done.